metron-analytics/metron-profiler-client/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,24 @@
+
+metron-profiler-client
+Copyright 2006-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product includes software developed for Orekit by
+CS Systèmes d'Information (http://www.c-s.fr/)
+Copyright 2010-2012 CS Systèmes d'Information
+
+                            The Netty Project
+                            =================
+
+Please visit the Netty web site for more information:
+
+  * http://netty.io/
+
+Copyright 2011 The Netty Project
+
+
+Curator Service Discovery
```

Comment:
Per clarification on https://issues.apache.org/jira/browse/LEGAL-234, you can omit the "[Bundled Software Name]\nCopyright XXXX-XXXY The Apache Software Foundation" as it is already covered by the earlier "This product includes software developed at The Apache Software Foundation (http://www.apache.org/)."